<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="top_wgt" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#top_wgt{background-color: rgb(170, 0, 0);}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_9" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QLabel" name="logo_lb">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>8</y>
            <width>24</width>
            <height>16</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;</string>
          </property>
          <property name="text">
           <string>logo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="fd_btn">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
	border: 1px solid gray;
}</string>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bk_btn">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
	border: 1px solid gray;
}</string>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>搜索视频、音乐、电台</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>249</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="logo_btn_2">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
	color:orange;
	border:none;
}
QPushButton:hover {
    color: white;
}</string>
            </property>
            <property name="text">
             <string>登录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="logo_btn_3">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: orange;
	border:none;
}
QPushButton:hover {
  	color: white;
   	
}</string>
            </property>
            <property name="text">
             <string>升级为VIP</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="email_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
	border:none;
}
QPushButton:hover {
	
	background-color: rgb(142, 32, 32);
}</string>
            </property>
            <property name="text">
             <string>邮箱</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="th_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
	border:none;
}
QPushButton:hover {
	
	background-color: rgb(142, 32, 32);
}</string>
            </property>
            <property name="text">
             <string>主题</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="set_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
	border:none;
}
QPushButton:hover {
	
	background-color: rgb(142, 32, 32);
}</string>
            </property>
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="min_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
	border:none;
}
QPushButton:hover {
	background-color: rgb(142, 32, 32);
}</string>
            </property>
            <property name="text">
             <string>min</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="max_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
	border:none;
}
QPushButton:hover {
	
	background-color: rgb(142, 32, 32);
}</string>
            </property>
            <property name="text">
             <string>max</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cls_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
	border:none;
}
QPushButton:hover {
	
	background-color: rgb(122, 0, 0);
}</string>
            </property>
            <property name="text">
             <string>close</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="main_wgt" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="left_wgt" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#left_wgt{
	background-color: rgb(217, 217, 217);
}
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>推荐</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>发现音乐</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>私人FM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_7">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>LOOK直播</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_8">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>视频</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_6">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>朋友</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>收藏</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_9">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>发现音乐</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_10">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>私人FM</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_11">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>LOOK直播</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_12">
               <property name="styleSheet">
                <string notr="true">QPushButton{
	text-align:left;
}</string>
               </property>
               <property name="text">
                <string>视频</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>创建的歌单</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="listView"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>收藏的歌单</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListView" name="listView_2"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="right_wgt" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#right_wgt{
	
	background-color: rgb(227, 227, 227);
}
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="styleSheet">
             <string notr="true">QTabWidget::pane{
    border-top: 1px solid rgba(69,104,220,0.3);
    background: transparent;
}

QTabBar::tab{
    min-width: 70px;
    min-height: 30px;
    background: transparent;
	color:blank;
}
QTabBar::tab:selected{
    color:rgb(69,104,220);
    background-color:rgba(240,245,255,0.8) ;
}

QTabBar::tab:hover{
    background-color:rgba(240,245,255,0.8) ;
}
</string>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="tabBarAutoHide">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="tab">
             <attribute name="title">
              <string>个性推荐</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>歌单</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>排行榜</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>主播电台</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>歌手</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_6">
             <attribute name="title">
              <string>最新音乐</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
